stunner/stunner.go

```diff
@@ -228,29 +253,36 @@ func (s *Stunner) sendSTUN(ctx context.Context, server string) error {
 	}
 	if addr.IP == nil {
 		if s.OnlyIPv6 {
-			return fmt.Errorf("cannot resolve any ipv6 addresses for %s, got: %v", server, ipAddrs)
+			return nil, fmt.Errorf("cannot resolve any ipv6 addresses for %s, got: %v", server, ipAddrs)
 		}
-		return fmt.Errorf("cannot resolve any ipv4 addresses for %s, got: %v", server, ipAddrs)
+		return nil, fmt.Errorf("cannot resolve any ipv4 addresses for %s, got: %v", server, ipAddrs)
 	}
+	return addr, nil
+}
 
-	txID := stun.NewTxID()
-	req := stun.Request(txID)
-	s.addTX(txID, server)
-	_, err = s.Send(req, addr)
+func (s *Stunner) sendPackets(ctx context.Context, server string) error {
+	addr, err := s.serverAddr(ctx, server)
 	if err != nil {
-		return fmt.Errorf("send: %v", err)
+		return err
 	}
-	return nil
-}
 
-var retryDurations = []time.Duration{
-	100 * time.Millisecond,
-	100 * time.Millisecond,
-	100 * time.Millisecond,
-	200 * time.Millisecond,
-	200 * time.Millisecond,
-	400 * time.Millisecond,
-	800 * time.Millisecond,
-	1600 * time.Millisecond,
-	3200 * time.Millisecond,
+	const maxSend = 2
```

Comment:
only 2?

Reply:
This is to limit damage for now until the next step where it more intelligently collects multiple STUN pings (as replied to Dave above)

Plus I think even 1 would be fine for our use cases.